ush/ocnice_extractvars.sh

```diff
@@ -21,15 +21,23 @@ comout_rfcst_prod_ocnice=${6}
 
 [[ -d "${subdata}" ]] || mkdir -p "${subdata}"
 
-for (( nh = FHMIN_GFS; nh <= FHMAX_GFS; nh = nh + fhout_ocnice )); do
+for (( nh = FHMIN_GFS + fhout_ocnice; nh <= FHMAX_GFS; nh = nh + fhout_ocnice )); do
   fnh=$(printf "%3.3d" "${nh}")
 
   if [[ ${component_name} == "ocn" ]]; then
-    infile=${COMIN_OCEAN_NETCDF}/${datares}/${RUN}.ocean.t${cyc}z.${datares}.f${fnh}.nc
+    if [[ "${datares}" == "native" ]]; then
+      infile=${COMIN_OCEAN_HISTORY}/${RUN}.ocean.t${cyc}z.${fhout_ocnice}hr_avg.f${fnh}.nc
+    else
+      infile=${COMIN_OCEAN_NETCDF}/${datares}/${RUN}.ocean.t${cyc}z.${datares}.f${fnh}.nc
```

Comment:
```suggestion
      infile="${COMIN_OCEAN_NETCDF}/${datares}/${RUN}.ocean.t${cyc}z.${datares}.f${fnh}.nc"
```
What is the value of `datares` supposed to be here? This is the interpolated grid correct? Should this be `grid_res`; `1p00`, 0p25` etc?

Reply:
For the reforecast, we aim to have `datares` be `1p00` for ocean and `native` (native resolution) for ice. This means we are extracting ocean products with an interpolated grid from `COMIN_OCEAN_NETCDF`. We are extracting ice products with a native grid from `COMIN_ICE_HISTORY`. We are extracting ice variables from `COMIN_ICE_HISTORY` because all the variables that we need to extract are available there.